### PR TITLE
nav refresh

### PR DESCRIPTION
**Pull request by owocki**
i disliek the current nav.  there is too much going on and it's hard to find things once youre inside the site.

# two things

## one - extenral vs internal nav

this PR factors the nav (previously only divided on logged in / logged out) along another axis:

* internal -- the actual tools of the application
* external -- the brochureware of the application

combined with the previous parts of the nav (logged in / logged out), this means there are four sections in the nav

* internal logged in
* external logged in
* internal logged out
* external logged out

screen shot 2018-06-10 at 5 45 18 pm

## two - tools dropdown

4b5b8cfc516d0febaa6f19e010dea034a2270432 also introduces the idea of a 'tools' dropdown menu', which looks like this: 

screen shot 2018-06-10 at 7 24 36 pm

Review by @PixelantDesign @thelostone-mc @SaptakS (the frontend team)

### CODE REVIEW

File: app/assets/v2/css/base.css [hunk 1]
```diff
@@ -30,8 +30,13 @@ span {
...
 }
 
-.navbar-nav .nav-link {
+.navbar-dark .navbar-nav .nav-link {
   border-radius: 2px;
+  color: #7c78a4;
+}
+.interior, .interior a,
+.nav-link.interior, .nav-link.interior a {
+  color: white !important;
```
[the remark this answers is not in the document]
Comment (SaptakS) [reply app/assets/v2/css/base.css]: Also seems to affect the radio options in the issue explorer page.

**Comment by mbeacom** [comment]
Latest deployed to stage.

**Comment by PixelantDesign** [comment]
I think we *definitely* need a nav restructure, It's awesome you're taking the initiative to get it going!

* Grouping Login & Fund Issue together. This might make it seem like contributors should click here to login, when it's really targeted towards funders.
* Changing the navigation based on relevancy is okay but what's worth understanding is how users understand and what they think of the tools listed. 
* Some items link out to other sites - we can set expectations by grouping these things together and showing a link to external site icon.
* Mission and About could probably be in the same category. 

I think the nav is **highly** important, we don't want to change it on users too many times as it could cause frustration. It deserves some light research - card sorting and tree testing with users.  

Tor a gut check - we could run what's on staging through a handful of funders and a handful of contributors. thoughts?

**Comment by owocki** [comment]
from my convo with @PixelantDesign on slack

ok so heres what im gonna do (when i finally get to the bottom of my bottomless notifications / emails)..

ill pare back the PR to be a less dramatic change
- undo the login and fund issue thing
- undo the tools change

**Pull request closed without merging**